mmv1/third_party/terraform/data_sources/data_source_google_service_account_jwt.go

```diff
@@ -29,14 +30,14 @@ func DataSourceGoogleServiceAccountJwt() *schema.Resource {
 			"target_service_account": {
 				Type:         schema.TypeString,
 				Required:     true,
-				ValidateFunc: validateRegexp("(" + strings.Join(PossibleServiceAccountNames, "|") + ")"),
+				ValidateFunc: transport_tpg.ValidateRegexp("(" + strings.Join(PossibleServiceAccountNames, "|") + ")"),
```

Comment:
Does ValidateRegexp really belong in the transport package?

Reply:
Good catch. It does not, actually. I will move it to the verify package.